pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/prometheus/monitor.go

```diff
@@ -47,6 +48,12 @@ kind: ServiceMonitor
 metadata:
   labels:
     control-plane: controller-manager
+    app.kubernets.io/name: servicemonitor
+    app.kubernetes.io/instance: controller-manager-metrics-monitor
+    app.kubernetes.io/component: metrics
+    app.kubernetes.io/created-by: {{ .ProjectName }}
+    app.kubernetes.io/part-of: {{ .ProjectName }}
+    app.kubernetes.io/managed-by: prometheus
```

Comment:
```suggestion
    app.kubernetes.io/managed-by: kustomize
```

Reply:
resolved in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/de5ad3beb0988a49f533ee8c1a8c87fd82e7bf02